lib/streamlit/server/Server.py

```diff
@@ -571,6 +576,9 @@ class _BrowserWebSocketHandler(tornado.websocket.WebSocketHandler):
     def initialize(self, server):
         self._server = server
         self._session = None
+        # Reading xsrf_token will set the xsrf cookie
+        # https://www.tornadoweb.org/en/stable/guide/security.html#cross-site-request-forgery-protection
```

Comment:
nitpick: paste the doc snippet to avoid context-switch when reading the code:

```suggestion
        # The XSRF cookie is normally set when xsrf_form_html is used, but in a pure-Javascript application
        # that does not use any regular forms we just  need to read the self.xsrf_token manually to set the
        # cookie as a side effect).
        # See https://www.tornadoweb.org/en/stable/guide/security.html#cross-site-request-forgery-protection
        # for more details.
```